package.json

```diff
@@ -37,7 +37,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run precommit"
```

Comment:
I think this line should stay, since the `precommit` script does more than just run lint.